en/faq/deployment-azure-portal

```diff
@@ -0,0 +1,76 @@
+#How to deploy on Azure Portal
+
+If you are using Azure DevOps and let the build pipeline do it's work you and want to store artefacts. All the files that 
+start with a `.` you have to explicitly copy them to the artifact folder. Then you can create a Artifact Archive and 
+download it after in your Release Deployment. 
+
+## Running the webserver
+For Azure Portal you will need a web.config file. If not supplied it will create one itself. This one won't work for Nuxt though.
```

Comment:
```suggestion
For Azure Portal you will need a `web.config` file. If not supplied, it will create one itself. This one **won't work for Nuxt** though.
```